### PR TITLE
Fix nonvc noetic

### PR DESCRIPTION
**Pull request by daricht**
Fixed the novnc Rviz and added documentation.

### CODE REVIEW

File: ros-noetic-moveit/Dockerfile
Comment (gidodielemans) [inline ros-noetic-moveit/Dockerfile:1]: needs change to augmentedfabricationlab/ros-noetic-base

File: ros-noetic-novnc/docker-compose.yml
Comment (gidodielemans) [inline ros-noetic-novnc/docker-compose.yml:1]: don't think this belongs in this repository

File: ros-noetic-novnc/Novnc Docker hierarchy.pdf
Comment (gidodielemans) [inline ros-noetic-novnc/Novnc Docker hierarchy.pdf:1]: Not sure this needs to be here, could be explained in the readme if necessary

**Review by gidodielemans — CHANGES_REQUESTED** [review]
Please adjust based on the comments :)

File: ros-noetic-moveit/Dockerfile
Comment (daricht) [reply ros-noetic-moveit/Dockerfile:1]: see commit b2467da6b585a4df7fe2f6f1a9d91dddd078fc53 i just made to fix this

**Pull request closed without merging**